eventGenerator/generator.js

```diff
@@ -0,0 +1,190 @@
+import fs from 'fs';
+import uuid from 'uuid';
+import resolveStorage from 'resolve-storage';
+import storageDriver from 'resolve-storage-file';
+
+import eventTypes from '../common/events/index';
+import HNServiceRest from './services/HNServiceRest';
+
+const MAX_STORY_COUNT = 50;
+
+let events = [];
+let users = {};
+let storyIds = [];
+
+const dbPath = './storage.json';
+let canceled = false;
+
+const isCanceled = () => canceled;
```

Comment:
Is it really necessary to use function but not a variable?

Reply:
fixed